client/v2/CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v2.0.0-beta.2] - 2023-02-19
```

Comment:
I want to get these in before monday: https://github.com/cosmos/cosmos-sdk/pull/19377 / https://github.com/cosmos/cosmos-sdk/pull/19376